install/kubernetes/cilium/templates/clustermesh-apiserver/tls-certmanager/server-secret.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
   namespace: {{ .Release.Namespace }}
 spec:
   issuerRef:
-    {{- include "clustermesh-apiserver-generate-certs.certmanager.issuer" . | nindent 4 }}
+    {{- toYaml .Values.clustermesh.apiserver.tls.auto.certManagerIssuerRef | nindent 4 }}
```

Comment:
as this PR is marked as backport, do you foresee any issue with the upgrade path (e.g. any downtime) ?

Reply:
Same as above